src/constants.js

```diff
@@ -31,4 +33,23 @@ export const permissionResponses = {
 export const imageSourceTypes = {
     CAMERA: 'camera',
     CAMERA_ROLL: 'camera_roll'
+};
+
+export const facilityIconDetails = {
+    restrooms: {name: 'human-male-female', type: 'material-community'},
+    potable_water: {name: 'cup-water', type: 'material-community'},
+    showers: {name: 'shower', type: 'font-awesome'},
+    metal_firepit: {name: 'md-bonfire', type: 'ionicon'},
+    tables: {name: 'chair-school', type: 'material-community'},
+    none: {name: 'not-interested'}
+};
+
+export const featureIconDetails = {
+    evergreens: {name: 'tree', type: 'font-awesome'},
+    color_changing_trees: {name: 'tree', type: 'entypo'},
+    lake_view: {name: 'weather-sunset', type: 'material-community'},
+    river_view: {name: 'waves', type: 'material-community'},
+    secluded: {name: 'user-secret', type: 'font-awesome'},
+    mountain_views: {name: 'mountains', type: 'foundation'},
+    easy_access: {name: 'road', type: 'font-awesome'},
```

Comment:
These two constants hold our customization for our icons as they match up with our facilities and features.